Makefile

```diff
@@ -214,6 +214,20 @@ helm-docs: $(HELM_DOCS) ## Use helm-docs to generate documentation from helm cha
 		-t ../../../Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md \
 		-t ../../../Documentation/Helm-Charts/_templates.gotmpl
 
+check-helm-docs:
```

Comment:
Since `make docs` also builds the helm docs, do we still need this separate `check-helm-docs`? Seems like it's already tested by `make check-docs`?

Reply:
@travisn  wrote: 
> Since `make docs` also builds the helm docs, do we still need this separate `check-helm-docs`? Seems like it's already tested by `make check-docs`?


good point. I thought about it but decided to keep the separate targets because the `check-helm-docs` target has a more specific error message, pointing to a helm-docs specific section in the guide.